integration/cluster_proxy.go

```diff
@@ -56,13 +56,14 @@ func toGRPC(c *clientv3.Client) grpcAPI {
 	kvp, kvpch := grpcproxy.NewKvProxy(c)
 	wp, wpch := grpcproxy.NewWatchProxy(c)
 	lp, lpch := grpcproxy.NewLeaseProxy(c)
+	mp := grpcproxy.NewMaintenanceProxy(c)
```

Comment:
should you have a mpch store in `grpcClientProxy` struct like rest of proxy does? It is useful in [`proxyCloser.Close()`](https://github.com/coreos/etcd/blob/master/integration/cluster_proxy.go#L79) where it waits for all the proxies to shutdown.